ios/RNSScreenStackHeaderSubview.mm

```diff
@@ -68,9 +71,9 @@ - (instancetype)initWithFrame:(CGRect)frame
 
 #pragma mark - RCTComponentViewProtocol
 
-- (void)prepareForRecycle
+- (RCTImageLoader *)imageLoader
 {
-  [super prepareForRecycle];
+  return imgLoader;
 }
```

Comment:
Naaa, even if the recycling is disabled, please do not mix up all the changes in this single PR. Podfile locks, this. Let's clean this up before merging.